test/linalg/qr.jl

```diff
@@ -144,3 +165,10 @@ let
     Q=full(qrfact(A)[:Q])
     @test vecnorm(A-Q) < eps()
 end
+
+qrpa = qrfact(areal,Val{true})
+qrpa = convert(QRPivoted{BigFloat},qrpa)
+#@test A_ldiv_B!(qrpa,zeros(BigFloat,n)) ≈ zeros(BigFloat,n)
```

Comment:
They are commented out because I wanted to make sure `\` was calling the right thing. It is, so I'll remove the comments.